bentoml/configuration/default_bentoml.cfg

```diff
@@ -20,6 +20,7 @@
 debug = false
 usage_tracking = true
 bentoml_deploy_version = {BENTOML_VERSION}
+docker_base_image = continuumio/miniconda3:4.8.2
```

Comment:
consider renaming this to `default_docker_base_image`? it could be more clear to the user when both this config and the `@env` parameter, the image specified in `@env` parameter will be picked up

Reply:
Got it. I will create a PR soon.

Reply:
Please check https://github.com/bentoml/BentoML/pull/694.